assets/js/components/NewChecksResults/CheckResultOutline.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {
+  EOS_COLLOCATION,
+  EOS_DESKTOP_WINDOWS,
+  EOS_KEYBOARD_ARROW_RIGHT,
+} from 'eos-icons-react';
+import classNames from 'classnames';
+
+const TARGET_NODE = 'TARGET_NODE';
+const TARGET_CLUSTER = 'TARGET_CLUSTER';
+
+const isExpect = ({ type }) => type === 'expect';
+const isExpectSame = ({ type }) => type === 'expect_same';
+
+function CheckResultOutline({
+  checkId,
+  agentsCheckResults,
+  expectationResults,
+  clusterName,
+}) {
+  const expectExpectationsCount = expectationResults.filter(isExpect).length;
```

Comment:
We could have this `expectExpectationsCount` inside the next `map` function I guess

Reply:
Not really I guess, we don't need that expression evaluated for each mapped element :thinking: 
What am I missing?